app/views/projects/show.html.erb

```diff
@@ -1,33 +1,86 @@
-<div class="container mx-auto">
-  <div class="mt-5">
-    <div class="mb-3">
-      <%= @project.title %>
+<div class="page-wrapper">
+  <div class="page-header d-print-none text-white">
+    <div class="container-xl">
+      <div class="row g-2 align-items-center">
+        <div class="col">
+          <div class="page-pretitle">
+            Project overview
+          </div>
+          <h2 class="page-title">
+            <%= link_to @project.title, project_path(@project) %>
+          </h2>
+        </div>
+        <div class="col-auto ms-auto d-print-none">
+          <div class="btn-list">
+            <span class="dropdown">
+              <a href="#" class="btn btn-primary dropdown-toggle align-text-top show" data-bs-boundary="viewport" data-bs-toggle="dropdown">
+                <%= inline_svg_pack_tag('static/@tabler/icons/icons/plus.svg') %>
+                New
+              </a>
+              <div class="dropdown-menu dropdown-menu-end" data-popper-placement="bottom-end" style="position: absolute; inset: 0px 0px auto auto; margin: 0px; transform: translate3d(0px, 29.5px, 0px);">
+                <%= link_to 'Account', new_project_account_path(@project) %>
+              </div>
+            </span>
+            <% if policy(@project).edit? %>
+              <a class="btn btn-primary" href="#">
+                <%= inline_svg_pack_tag('static/@tabler/icons/icons/pencil.svg') %>
+                Edit
+              </a>
+            <% end %>
+          </div>
+        </div>
+      </div>
     </div>
   </div>
-
-  <div>
-    <h4>Maintainers</h4>
-    <%= link_to 'Add Maintainer', edit_project_maintainers_path(@project), class: 'btn btn-primary' %>
-      <ul class="list-group list-group-flush">
-        <% @project.maintainers.each do |maintainer| %>
-          <li class="list-group-item"><%= link_to maintainer.first_name, maintainer, class: 'nav-link' %></li>
-        <% end %>
-      </ul>
-  </div>
-
-  <div>
-    <h4>Accounts</h4>
-    <% if policy(@project).edit? %>
-      <%= link_to 'Add new Account', new_project_account_path(@project), class: 'btn btn-primary' %>
-    <% end %>
-    <% if @project.accounts.any? %>
-      <%= link_to 'Accounts', project_accounts_path(@project), class: 'btn btn-primary' %>
-    <% else %>
-      <p>No accounts could be found for this project.</p>
-    <% end %>
+  <div class="page-body">
+    <div class="container-xl">
+      <div class="row row-cards">
+        <div class="col-sm-6 col-lg-3">
+          <%= render 'projects/counter_overview', icon_path: 'static/@tabler/icons/icons/key.svg' do %>
+            <%= link_to "#{@project.accounts.count} Users", '#' %>
```

Comment:
Hibás a szöveg, nem user-eket számolunk, hanem accountokat, ráadásul sehova nem mutat a link, be kellene kötni a meglévő accounts listázó oldalt